vignettes/renv.Rmd

```diff
@@ -0,0 +1,47 @@
+---
+title: "renv"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{renv}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+```{r setup}
+library(riskassessment)
+```
+
+# Overview
+
+The `renv` package provides project-local R dependency management to your projects.  
+It helps manage library paths (and other project-specific state) to help isolate your project’s R dependencies.  
+The `renv` package helps you create **r**eproducible **env**ironments for your R projects.  
+It will make your R packages more **isolated**, **portable** and **reproducable**.
+
+## How to use it
+
+First one has to initialize a new project-local environment with a private R library - which can be done via `renv::init()`.
+Afterwards, work in the project as usual and save the current state of the project library with `renv::snapshot()`.
+This result will be saved into a lockfile - named `renv.lock`.
+Now, continue working on your project and call `renv::snapshot()` again, in order to update the project state - or with `renv::restore()`, revert the previous state if you have encountered problems while developing.
+
+## How to ignore files in the lock file
+
+By default `renv` ignores all files stated in the `.gitignore` file. If you want to extend that further, you can use a `.renvignore` file - which is specified similar to the `.gitignore` file.
+
```

Comment:
It may be worth adding logic here around the dev team philosophy of upgrading or changing the existing `lockfile` once it is stable. 

My suggestion would be to never upgrade once stable unless there is a very compelling reason to do so (e.g. new feature we can't live without).

Reply:
That sounds good to me. @Xyarz, could you add a sentence or two that summarizes what Andrew has stated above?

Reply:
added in [a5a96fa](https://github.com/pharmaR/riskassessment/pull/315/commits/a5a96fa7644de7e69f087394effc1ab045739b32)